docs/installguide/system_requirements.rst

```diff
@@ -1,12 +1,87 @@
 System requirements
 ===================
 
+Operating systems
+-----------------
+
+ * Windows XP, Vista, 7, 8, 10
```

Comment:
Release 0.16 from last March was tested and working on XP, with known issues like videos only playing in Chrome (not in FF nor IE) - it was a VideoJS issue if I recall well. 

MS dropped XP from available list of VMs at modern.ie, so that's become an issue for comprehensive testing too. At this point we must evaluate if it's viable to continue supporting XP any longer considering it's EOL status on one hand, and the number of deployments we have which will not be able to update if we decide to stop supporting it. Maybe @jamalex can chime in?